### ISSUE TITLE
Can't get "multipage" (Poll) sample(s) to work

### ISSUE THREAD

**Issue by Languafe** [SharePoint/sp-dev-fx-webparts#333]
Hey, I'm trying to get the hang of this multipage sample (https://github.com/SharePoint/sp-dev-fx-webparts/tree/master/samples/react-multipage), but the readme is telling me to copy a renamed workbench.html to workbench.aspx to a document library in my site?! SharePoint Online doesn't allow me to just do this and then view the aspx page as far as I know - or can someone tell me how?

I have the workbench available at /_layouts/workbench.aspx under all sites in SPO don't I? This is shown in the getting started tutorials if I recall correctly.

So anyway when I try to use the workbench.aspx in my SPO site I find the web part to add it, but then it fails. I am seeing an Uncaught Error: Sys.InvalidOperationException in dev tools console as soon as I open the workbench (unrelated to the sample, I get it). I am still able to use the workbench to add the "Poll" webpart, but then as soon as I do there's another error "Minified React" error 130.

All suggestions appreciated, thanks in advance.

**Comment by waldekmastykarz** [comment]
This sample has been built using an older version of the SharePoint Framework which could be the reason why it's not working in SPO.

1. Workbench: correct, you don't have to upload workbench anymore. In the past there was no hosted version, which is why you had to do it, but now there is, so this step is obsolete.
1. This error is likely related to the sample using an older version of SPFx. Ideally, you would update the sample to the latest version of SPFx (either on-prem on online) and try running it then, which should work as expected.

**Comment by Languafe** [comment]
Thanks for the quick reply. I just realised you're the author of the sample!

I tried, optimistically, to just changing the versions of the dependencies in package.json, but now getting overrun with errors. I'm not overly comfortable with the various parts in this solution - TypeScript, React and the tooling is pretty new to me.

I guess I could just start with a brand new web part project using the yeoman generator and see if I'm able to build it from scratch.
